scripts/build_utils.sh

```diff
@@ -874,6 +874,7 @@ ceph_build_args_from_flavor() {
         ;;
     crimson)
         CEPH_EXTRA_RPMBUILD_ARGS="--with seastar"
+        CEPH_EXTRA_RPMBUILD_ARGS="${CEPH_EXTRA_RPMBUILD_ARGS} --with crimson"
```

Comment:
This construct looks weird / confusing. Isn't `+=` working here?

Reply:
I tried `+=` first but I saw already existing users here that go with this approach so I preferred keeping it that way. This should be removed once the PR mentioned is merged.